src/app/models/Subject.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Subject {
+  _id: any;
```

Comment:
perchè l'id è di tipo any? è una stringa.

Reply:
perché avevo in testa che fosse Objectid di mongodb